lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
-  "version": "0.0.0",
+  "version": "3.0.0",
```

Comment:
?

Reply:
Lerna has no issues resolving downgrades, should be fine.

![image](https://github.com/user-attachments/assets/eb9ddf8c-33d9-4b26-be23-6ab09355ce4c)